### PR TITLE
feat(@wallet): 5 chars min account keypair name

### LINKED ISSUE #13384: Validate account name and keypairs that are under 5 characters
New validation states have been added to the various flows listed below to prevent the user from entering an account or keypair name that is <5 characters. This change was requested originally by @saledjenic and has been completed as part of a sweep of backlog items. 

**Flows where this validation is now featured:**

**1. Editing an existing account**

Screenshot 2024-02-01 at 4 34 17 PM

Figma:
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=21039-220481&mode=design&t=2O68lxNGG9g1b1tx-4

--

**2. Rename keypair (Wallet Settings)**

Screenshot 2024-02-01 at 4 28 20 PM

Figma:
https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=23264-284147&mode=design&t=i8YMYAMFRqAFgJ3v-4

--

**3. Add new keypair via seed phrase**

Screenshot 2024-02-01 at 4 27 22 PM

Figma:
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=21240-102220&mode=design&t=2O68lxNGG9g1b1tx-4

--

**4. Add new account via private key**

Screenshot 2024-02-01 at 4 30 47 PM

Figma:
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=21240-102386&mode=design&t=2O68lxNGG9g1b1tx-4

--

5. Generate new keypair - naming the new keypair
(Validation same as others and not shown again in design as would be duplicative)

### PR DESCRIPTION
**Pull request by alaibe**
fixes #13384

### CODE REVIEW

File: ui/app/AppLayouts/Profile/popups/RenameAccontModal.qml [hunk 1]
```diff
@@ -69,15 +69,15 @@ StatusModal {
             }
             validators: [
                 StatusMinLengthValidator {
-                    errorMessage: qsTr("You need to enter an account name")
-                    minLength: 1
+                    errorMessage: qsTr("Account name must be at least %1 characters").arg(Constants.addAccountPopup.keyPairAccountNameMinLength)
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/popups/RenameAccontModal.qml:72]:
```suggestion
                    errorMessage: qsTr("Account name must be at least %n character(s)", "", Constants.addAccountPopup.keyPairAccountNameMinLength)
```

Comment (caybro) [reply ui/app/AppLayouts/Profile/popups/RenameAccontModal.qml:72]: Aka proper plural missing

File: ui/imports/shared/popups/addaccount/states/EnterKeypairName.qml [hunk 2]
```diff
@@ -43,6 +44,13 @@ Item {
             onKeyPressed: {
                 root.store.submitPopup(event)
             }
+
+            validators: [
+                StatusMinLengthValidator {
+                    errorMessage: qsTr("Keypair name must be at least %1 characters").arg(Constants.addAccountPopup.keyPairAccountNameMinLength)
```
Comment (caybro) [inline ui/imports/shared/popups/addaccount/states/EnterKeypairName.qml:50]: Same here

File: ui/imports/shared/popups/addaccount/states/Main.qml [hunk 3]
```diff
@@ -126,8 +126,8 @@ Item {
                 }
                 validators: [
                     StatusMinLengthValidator {
-                        minLength: 1
-                        errorMessage: Utils.getErrorMessage(accountName.errors, qsTr("wallet account name"))
+                        errorMessage: qsTr("Account name must be at least %1 characters").arg(Constants.addAccountPopup.keyPairAccountNameMinLength)
```
Comment (caybro) [inline ui/imports/shared/popups/addaccount/states/Main.qml:129]: Same here

File: ui/imports/shared/popups/common/EnterPrivateKey.qml [hunk 4]
```diff
@@ -176,6 +176,13 @@ Item {
                 onKeyPressed: {
                     root.store.submitPopup(event)
                 }
+
+                validators: [
+                    StatusMinLengthValidator {
```
Comment (caybro) [inline ui/imports/shared/popups/common/EnterPrivateKey.qml:181]: And here

File: ui/imports/shared/popups/common/EnterSeedPhrase.qml [hunk 5]
```diff
@@ -88,6 +88,13 @@ Item {
                     root.store.addAccountModule.newKeyPairName = text
                 }
 
+                validators: [
+                    StatusMinLengthValidator {
+                        errorMessage: qsTr("Keypair name must be at least %1 characters").arg(Constants.addAccountPopup.keyPairAccountNameMinLength)
```
Comment (caybro) [inline ui/imports/shared/popups/common/EnterSeedPhrase.qml:93]: Here too

Comment (caybro) [reply ui/imports/shared/popups/common/EnterSeedPhrase.qml:93]: Given the amount of repetition, extract the whole Validator somewhere? :)

**Review by caybro — CHANGES_REQUESTED** [review]

**Review by stefandunca — APPROVED** [review]
Good to go after checking @caybro's recommendations

File: ui/imports/shared/popups/common/EnterSeedPhrase.qml (hunk 5 above)
Comment (alaibe) [reply ui/imports/shared/popups/common/EnterSeedPhrase.qml]: I'd vote for a AccountNameInput and KeypairNameInput.
Many other things are repeated
If you agree i ll create the issue

**Review by caybro — APPROVED** [review]

**Pull request merged**